apps/prairielearn/src/pages/instructorLoadFromDisk/instructorLoadFromDisk.ts

```diff
@@ -23,7 +27,13 @@ async function update(locals: Record<string, any>) {
 
   serverJob.executeInBackground(async (job) => {
     let anyCourseHadJsonErrors = false;
-    await async.eachOfSeries(config.courseDirs || [], async (courseDir, index) => {
+
+    // Merge the list of courses in the config with the list of courses in the database.
```

Comment:
Add a comment that we are using a `Set()` so that we won't double-count courses that are both in the DB and in the config (especially on a second pass)?